tests/env.sh

```diff
@@ -1,7 +1,9 @@
-export BROKER_URL=localhost//guest:guest@rabbitmq//
-export DATABASE_URL=mysql://root@localhost:3306/treeherder
+export BROKER_URL=amqp://guest:guest@localhost:5672//
```

Comment:
I grabbed this value from the [PR](https://github.com/mozilla/treeherder/pull/4901/files#diff-072b4ed37863351df037e267b1312a01L9) that moved us to Docker.

Without this value being set correctly one of the last tests to execute (`test_consumers.py`) gets stuck without output.